tasks/setup_env.yml

```diff
@@ -18,6 +18,12 @@
     - ca-certificates
   when: ansible_os_family in [ "Debian" ]
 
+- name: "[ENV] - Upgrade all packages to latest version."
+  apt: upgrade=dist force_apt_get=yes
```

Comment:
FQDN and ansible-lint as it was copied from the other file.

Reply:
Have installled ansible--flint and modified files to comply. Two warnings still exist, rest is resolved. The first warning seems to be related to flint, while the second cannot be avoided.